usagereportsv4/usage_reports_v4.go

```diff
@@ -1436,17 +1436,32 @@ type InstanceUsage struct {
 	// The ID of the resource instance.
 	ResourceInstanceID *string `json:"resource_instance_id" validate:"required"`
 
+	// The name of the resource instance.
+	ResourceInstanceName *string `json:"resource_instance_name,omitempty"`
+
 	// The ID of the resource.
 	ResourceID *string `json:"resource_id" validate:"required"`
 
+	// The name of the resource.
+	ResourceName *string `json:"resource_name,omitempty"`
+
 	// The ID of the resource group.
 	ResourceGroupID *string `json:"resource_group_id,omitempty"`
 
+	// The name of the resource group.
+	ResourceGroupName *string `json:"resource_group_name,omitempty"`
+
 	// The ID of the organization.
 	OrganizationID *string `json:"organization_id,omitempty"`
 
+	// The name of the organization.
+	OrganizationName *string `json:"organization_name,omitempty"`
+
 	// The ID of the space.
-	Space *string `json:"space,omitempty"`
+	SpaceID *string `json:"space_id,omitempty"`
```

Comment:
This particular change (i.e. changing the "space" property to be named "space_id") technically represents a breaking change.  For example, if an application is currently referencing the "Space" field, it will fail to compile when the app moves up to the new version of the Go SDK.   In this particular case (for this service and the fact that it is a response schema), that might not be a critical problem, but technically it's a breaking change :)

Reply:
The issue here is the correct field is "space_id" as there is no field called "space" from the API response.

Reply:
Ah, ok great! In that case we're correcting something that was incorrect before :)   No worries then.